src/wallet/wallet.cpp

```diff
@@ -585,8 +585,11 @@ bool CWallet::EncryptWallet(const SecureString& strWalletPassphrase)
         Lock();
         Unlock(strWalletPassphrase);
 
-        // if we are using HD, replace the HD seed with a new one
-        if (auto spk_man = GetLegacyScriptPubKeyMan()) {
+        // If we are using descriptors, make new descriptors
```

Comment:
Descriptors don't contain private keys, so maybe make it more clear that we're generating a new seed, and thus need new descriptors for that.

Reply:
Done